tests/e2e/Calendar.spec.ts

```diff
@@ -0,0 +1,43 @@
+import HomePage from "./pages/home.page";
+import OnClickEventPage from "./pages/onClick.page";
+import OnCreateEventPage from "./pages/onCreate.page";
+
+describe("Calendar", () => {
+    it("renders month view", () => {
+        HomePage.open();
+        HomePage.calendar.waitForExist();
+        HomePage.monthView.waitForExist();
+        const doesExist = HomePage.monthView.isExisting();
+
+        expect(doesExist).toBeTruthy();
+    });
+
+    it("renders day view", () => {
+        HomePage.dayViewButton.click();
+        HomePage.dayView.waitForExist();
+        const doesExist = HomePage.dayView.isExisting();
+
+        expect(doesExist).toBeTruthy();
+    });
+
+    it("should excecute an action on clicking an event", () => {
```

Comment:
`should execute an action when an event is clicked`

Reply:
There two different click events am testing